app/grandchallenge/components/models.py

```diff
@@ -176,16 +176,24 @@ class InterfaceKinds(set, Enum):
         from grandchallenge.components.models import INTERFACE_KIND_JSON_EXAMPLES
 
         print("Interface kinds that are images:")
+        print("")
```

Comment:
Maybe tidy this up a bit by using `\n` in the previous calls? It is also missing from `print("Interface kinds that are files:")`